src/Map/src/Bridge/Google/README.md

```diff
@@ -78,6 +80,63 @@ $googleOptions = (new GoogleOptions())
 // Add the custom options to the map
 $map->options($googleOptions);
 ```
+## Use cases
+
+Below are some common or advanced use cases when using a map.
+
+> **Note**:
+> As the UX Map component is new, there are not too many examples for the moment.
```

Comment:
```suggestion
> **Note**:
> As the UX Map component is new, there are not too many examples for the moment.
```

No need to add that. And README.md won't be the place were many examples would be added i think. More something for documentation and/or demos :)